usr/bin/nxdk-configure

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/sh
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+export CC=nxdk-cc
+export PKG_CONFIG=nxdk-pkg-config
```

Comment:
Lacks C++ support and might try to use other host tools or files, because autoconf is a bit of a trainwreck.